super-intel.c

```diff
@@ -7080,7 +7080,17 @@ get_devices(const char *hba_path)
 		}
 
 		dv = xcalloc(1, sizeof(*dv));
+		if (dv == NULL) {
```

Comment:
please see xcalloc implementation and you will realize that the code won't be executed because xcalloc will abort the operation.

If `err` is not used then I think that the error path should be removed.